src/update-feeds/update-feeds.ts

```diff
@@ -40,6 +40,9 @@ export const startUpdateFeedLoops = async () => {
 
       for (const providerName of Object.keys(providers)) {
         logger.debug(`Starting update feed loop`, { chainId, providerName });
+        // Run the update feed loop manually for the first time, because setInterval first waits for the given period of
+        // time.
+        void runUpdateFeed(providerName, chain, chainId);
```

Comment:
It seems there are no timeouts currently, so it's possible for these calls to overlap potentially quite a lot?